projects/laji/src/app/+project-form/project-form.component.html

```diff
@@ -69,11 +69,13 @@ <h5>
 </ng-template>
 
 <ng-template #badge let-badge>
+  <ng-container *ngIf="formPermissions$ | async as formPermissions">
       <span
-        *ngIf="(formPermissions$ | async)[badge.count]"
+        *ngIf="formPermissions$[badge.count]"
```

Comment:
Shouldn't this be `formPermissions` instead of `formPermissions$`?

Reply:
Yes - fixed